applications/cup2024-robot-motors/include/real/app_conf.hpp

```diff
@@ -20,3 +55,182 @@ constexpr float motor_lift_speed_pid_integral_limit = 500 / motor_lift_speed_pid
+constexpr float motor_lift_min_pose_mm = 0.0f;              ///< Lower travel limit (mm)
+constexpr float motor_lift_max_pose_mm = 200.0f;            ///< Upper travel limit (mm)
+
+/// @brief Limit switch GPIO pins.
+constexpr gpio_t lower_limit_switch_pin = GPIO_PIN(PORT_A, 6);  ///< Lower end-stop pin
+constexpr gpio_t upper_limit_switch_pin = GPIO_PIN(PORT_A, 4);  ///< Upper end-stop pin
+
+/// @brief Motor Lift pose PID controller
+static cogip::pid::PID motor_lift_pose_pid(
+    motor_lift_pose_pid_kp,
+    motor_lift_pose_pid_ki,
+    motor_lift_pose_pid_kd,
+    motor_lift_pose_pid_integral_limit
+    );
+
+/// @brief Motor Lift speed PID controller
+static cogip::pid::PID motor_lift_speed_pid(
+    motor_lift_speed_pid_kp,
+    motor_lift_speed_pid_ki,
+    motor_lift_speed_pid_kd,
+    motor_lift_speed_pid_integral_limit
+    );
+
+/// @brief Motor Lift MotorPoseFilterParameters
+static cogip::motion_control::MotorPoseFilterParameters motor_lift_pose_filter_parameters(
+    motor_lift_threshold,
+    lift_limits::max_deceleration_mm_per_period2
+);
+
+/// @brief Motor Lift PosePIDControllerParameters.
+static cogip::motion_control::PosePIDControllerParameters motor_lift_pose_pid_parameters(&motor_lift_pose_pid);
+
+/// @brief Motor Lift SpeedFilterParameters.
+static cogip::motion_control::SpeedFilterParameters motor_lift_speed_filter_parameters(
+    lift_limits::min_speed_mm_per_period,
+    lift_limits::max_speed_mm_per_period,
+    lift_limits::max_acceleration_mm_per_period2,
+    false,
+    lift_anti_blocking::speed_threshold,
+    lift_anti_blocking::error_threshold,
+    lift_anti_blocking::blocked_cycles_threshold
+    );
+
+/// @brief Motor Lift SpeedPIDControllerParameters.
+static cogip::motion_control::SpeedPIDControllerParameters motor_lift_speed_pid_parameters(&motor_lift_speed_pid);
+
+enum {
+    MOTOR_LIFT_ID = static_cast<uint8_t>(cogip::actuators::Enum::MOTOR_LIFT)
+};
+
+/// @brief Lifts Motor driver
+static cogip::motor::MotorDriverDRV8873 lifts_motor_driver(actuators_motors_params);
+
+/// @brief Motors
+static cogip::motor::MotorRIOT lift_motor(lifts_motor_driver, MOTOR_LIFT_ID);
+
+/// @brief Lift motor encoder
+static cogip::encoder::EncoderQDEC lift_motor_encoder(MOTOR_LIFT_ID,
+    cogip::encoder::EncoderMode::ENCODER_MODE_X4,
+    lift_motor_configuration::wheels_encoder_resolution);
+
+static cogip::localization::OdometerEncoderParameters lift_motor_odometer_params{
+    /* pulse_per_mm     */ lift_motor_configuration::pulse_per_mm,
+    /* reverse_polarity */ true
+};
+
+/// @brief Lift motor odometer
+static cogip::localization::OdometerEncoder lift_motor_odometer(lift_motor_odometer_params, lift_motor_encoder);
+
+// 1) Prepare generic motor parameters for the front lift actuator
+static const cogip::actuators::positional_actuators::MotorParameters lift_motor_params{
+    /* id                           */ cogip::actuators::Enum::MOTOR_LIFT,
+    /* default_timeout_ms           */ 2000u,
```

Comment:
This can be declared as a constant with explicit name. Same for other hardcoded values in this file

Reply:
Done.